src/coreclr/jit/importer.cpp

```diff
@@ -2529,6 +2529,30 @@ inline void Compiler::impEvalSideEffects()
     verCurrentState.esStackDepth = 0;
 }
 
+/*****************************************************************************
+ *
+ *  If the stack entry is a tree with side effects in it, assign that
+ *  tree to a temp and replace it on the stack with refs to its temp.
+ *  i is the stack entry which will be checked and spilled.
+ */
+
+inline void Compiler::impSpillSideEffect(bool spillGlobEffects, unsigned i DEBUGARG(const char* reason))
```

Comment:
Ok, so this was just a move of code to its own function.

Reply:
Yes, that way it could be reused more easily.